velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -554,46 +554,62 @@ std::optional<std::string> PrestoQueryRunner::toSql(
   return sql.str();
 }
 
-std::optional<std::string> PrestoQueryRunner::toSql(
-    const std::shared_ptr<const core::HashJoinNode>& joinNode) {
-  if (!isSupportedDwrfType(joinNode->sources()[0]->outputType())) {
-    return std::nullopt;
+static const std::string joinKeysToSql(
```

Comment:
nit: folly::join and we can drop const then..

Reply:
Done.